src/styles/record-details.scss

```diff
@@ -168,3 +168,9 @@
     margin-left: spacing.space(1);
   }
 }
+
+.tna-accordion__summary {
+  font-family: supria-sans-condensed, "Arial Narrow", sans-serif;
+  font-size: 32px;
+  font-weight: 500;
+}
```

Comment:
Let's remove this for now. I know it's in to match the design but I think we have to plan design changes to the accordion a bit more carefully.